cmd/gotk/utils.go

```diff
@@ -62,7 +62,12 @@ const (
 
 func (*Utils) execCommand(ctx context.Context, mode ExecMode, command string) (string, error) {
```

Comment:
Can you please change the signature to:
```go
 execKubectlCommand(ctx context.Context, mode ExecMode, arg ...string) (string, error)
```

Reply:
After the signature change we need to pass an args array, so the current string concatenation should be rewrite to array append.

Reply:
I've changed it

Reply:
Can you please rebase with main.

Reply:
done

Reply:
Should I squash it or smth?

Reply:
Yes please squash